test/verify/check-packagekit

```diff
@@ -61,9 +61,13 @@ class TestUpdates(NoSubManCase):
     def setUp(self):
         super().setUp()
 
-        # only the yum backend properly recognizes "enhancement" severity; apt
-        # does not have that metadata and PackageKit-dnf does not parse it
-        if self.backend == "yum":
+        pkcon_version = self.machine.execute("pkcon --version").strip()
+
+        # only the yum and PackageKit ≥ 1.2.4 dnf (https://github.com/PackageKit/PackageKit/issues/268) backends
+        # properly recognize "enhancement" severity; apt does not have that metadata
+        self.supports_severity = not (self.backend == "apt" or pkcon_version < "1.2.4")
```

Comment:
Not the best way how to compare versions - but since this is just test, that will do :)

Reply:
Right, I know -- my hope is that by the time we have an 1.2.10 (like in 7 years from now? :grin: ) we can just drop this special case completely.